cmd/promtool/main.go

```diff
@@ -549,10 +551,35 @@ func QueryLabels(url *url.URL, name string, p printer) int {
 		return 1
 	}
 
+	var (
+		minTime = time.Now().Add(-9999 * time.Hour)
+		maxTime = time.Now().Add(9999 * time.Hour)
+	)
+
+	var stime, etime time.Time
+
+	if start == "" {
+		stime = minTime
```

Comment:
what about 
```
stime := minTime
if start != "" {
// change stime
}
```
and same below?

Reply:
Thanks, I like this way. Much cleaner than before.